WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -26,6 +26,37 @@ final class ShippingLabelFormViewModel {
     private(set) var destinationAddress: ShippingLabelAddress?
     private(set) var packagesResponse: ShippingLabelPackagesResponse?
     private(set) var selectedPackageID: String?
+    var selectedPackage: ShippingLabelPackageSelected? {
```

Comment:
⚠️ I don't see this covered in `ShippingLabelFormViewModelTests`. Should we add a test for this? 😁